### ISSUE TITLE
Missing badges

### ISSUE THREAD

**Issue by Bashamega** [eddiejaoude/github-trending-repos#12]
Hello:)

I think you should integrate some badges like the discord server and the rating etc.

**Comment by eddiejaoude** [comment]
yes that would be great, I think we can copy it from the other repos in the Org
